openresty-build-tools/kong-ngx-build

```diff
@@ -427,14 +430,18 @@ main() {
         OPENRESTY_OPTS=(
           "--prefix=$OPENRESTY_PREFIX"
           "--with-cc-opt='-I$OPENSSL_INSTALL/include'"
-          "--with-pcre-jit"
           "--with-http_ssl_module"
           "--with-http_realip_module"
           "--with-http_stub_status_module"
           "--with-http_v2_module"
           "-j$NPROC"
         )
 
+        if [ ! "$PROCESSOR_TYPE" == "s390x" ]; then
+          # pcre does not compile on s390x
+          OPENRESTY_OPTS+=('--with-pcre-jit')
+        fi
+
```

Comment:
Add --with-pcre-jit if we're not building for s390x.

Reply:
Actually, --with-pcre-jit can be removed, at least for later versions of OpenResty that include nginx 1.16.0 (and maybe earlier).  https://github.com/openresty/openresty/issues/62#issuecomment-514360656.

Reply:
OpenResty 1.15.8.x is at nginx version 1.15.6, which may not allow --with-pcre-jit to be removed.
https://github.com/openresty/openresty/blob/1.15.8.x/util/package-nginx.sh

Reply:
Removed s390x-specific check to include --with-pcre-jit and instead include it for older versions of openresty.